docs/index.rst

```diff
@@ -191,37 +191,53 @@ and change the cache location. Caching persists between Astroquery sessions.
 If you know the service you are using has released new data recently, or if you believe you are
 not recieving the newest data, try clearing the cache.
 
+Service specific settings
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Astroquery cache location is divided by service, so each service's cache should be managed invidually,
-however whether the cache is active and the expiration time are controlled centrally through the
-astroquery ``cache_conf`` module. Astroquery uses the Astropy configuration infrastructure, information about
+The Astroquery cache location is specific to individual services,
+so each service's cache should be managed invidually.
+The cache location can be viewed with the following command
+(using `Simbad <simbad/simbad.rst>`_ as an example):
```

Comment:
cross link to the anchor, not the rst file.

Reply:
I don't know what you mean by this.

Reply:
each page has a sphinx reference, or at least should have. Use those, the RST is not a good reference.